website/docs/d/federated_settings_identity_provider.html.markdown

```diff
@@ -36,7 +36,9 @@ data "mongodbatlas_federated_settings_identity_provider" "identity_provider_ds"
 ## Argument Reference
 
 * `federation_settings_id` - (Required) Unique 24-hexadecimal digit string that identifies the federated authentication configuration.
-* `identity_provider_id` - (Required) Unique 20-hexadecimal digit string that identifies the IdP. 
+* `identity_provider_id` - (Required) Unique 20-hexadecimal or 24-hexadecimal digit string that identifies the IdP. 
+
+**WARNING:** Starting at terraform provider 1.16.0 the allowed format for `identity_provider_id` will only be 24-hexadecimal digit string. See more [here](../guides/1.15.0-upgrade-guide.html.markdown)
```

Comment:
will create a draft of the upgrade guide in a follow up PR

Reply:
@Zuhairahmed your input for these changes in documentation would be appreciated.